### PR TITLE
feat: add PyTurboGrid

### PR DESCRIPTION
**Pull request by RobPasMue**
As title says

### CODE REVIEW

**Review by MaxJPRey — APPROVED** [review]
LGTM.

**Comment by RobPasMue** [comment]
This is blocked until the license issue is solved on PyTurboGrid API package. See https://github.com/ansys-internal/ansys-api-turbogrid/pull/186

File: doc/source/api.rst [hunk 1]
```diff
@@ -153,6 +153,13 @@ PyAnsys packages API reference
       :text-align: center
       :class-title: pyansys-card-title
 
+    .. grid-item-card:: PyTurboGrid
+      :img-top: _static/thumbnails/pyturbogrid.png
+      :link: https://turbogrid.docs.pyansys.com/version/stable/api_reference/index.html
+      :text-align: center
+      :class-title: pyansys-card-title
+      :margin: 2 2 0 0
```
Comment (RobPasMue) [inline doc/source/api.rst:161]:
```suggestion

```

File: doc/source/examples.rst [hunk 2]
```diff
@@ -142,6 +142,13 @@ PyAnsys packages examples
       :text-align: center
       :class-title: pyansys-card-title
 
+    .. grid-item-card:: PyTurboGrid
+      :img-top: _static/thumbnails/pyturbogrid.png
+      :link: https://turbogrid.docs.pyansys.com/version/stable/examples/index.html
+      :text-align: center
+      :class-title: pyansys-card-title
+      :margin: 2 2 0 0
```
Comment (RobPasMue) [inline doc/source/examples.rst:150]:
```suggestion

```

File: doc/source/index.rst [hunk 3]
```diff
@@ -194,6 +194,15 @@ it is now a collection of many Python packages for using Ansys products through
 
       Pythonic interface to communicate with Ansys System Coupling
 
+    .. grid-item-card:: PyTurbogrid
+      :img-top: _static/thumbnails/pyturbogrid.png
+      :link: https://turbogrid.docs.pyansys.com/version/stable
+      :text-align: center
+      :class-title: pyansys-card-title
+      :margin: 2 2 0 0
```
Comment (RobPasMue) [inline doc/source/index.rst:202]:
```suggestion
```

File: doc/source/user_guide.rst [hunk 4]
```diff
@@ -145,6 +145,13 @@ PyAnsys packages user guides
       :text-align: center
       :class-title: pyansys-card-title
 
+    .. grid-item-card:: PyTurboGrid
+      :img-top: _static/thumbnails/pyturbogrid.png
+      :link: https://turbogrid.docs.pyansys.com/version/stable/getting_started/index.html
+      :text-align: center
+      :class-title: pyansys-card-title
+      :margin: 2 2 0 0
```
Comment (RobPasMue) [inline doc/source/user_guide.rst:153]:
```suggestion

```

**Pull request merged**